src/services/unlocker.ts

```diff
@@ -70,6 +71,11 @@ export default class Unlocker implements IUnlocker {
     const collect = this.collector.collect();
     const cells: IndexerCell[] = [];
 
+    // Refresh the cache by fetching the latest compatible xUDT list from the specified URL.
+    // The default URL is:
+    // https://raw.githubusercontent.com/utxostack/typeid-contract-cell-deps/main/compatible-udt.json
+    await CompatibleXUDTRegistry.refreshCache();
```

Comment:
https://github.com/utxostack/rgbpp-sdk/blob/8c6705ac180ae0b0b0eac71c95e74d44bc493f2d/packages/ckb/src/utils/cell-dep.ts#L210-L227 will handle the `CompatibleXUDTRegistry` cache automatically.

Reply:
Removed in https://github.com/utxostack/btc-assets-api/pull/234/commits/fcd7ad6f315a32f693e00fecafb7125d552737a7